docs/coverage-reporter/index.md

```diff
@@ -443,3 +447,15 @@ Follow these instructions to validate that your coverage setup is working correc
     -   Branch name and commit SHA-1 hash corresponding to the CI/CD output
     -   Test coverage report that you're uploading to Codacy
     -   Any other relevant information or screenshots of your setup
+
+## 4. Enabling the Git provider integration {: id="enabling-git-provider-integration"}
+
+After validating that the coverage setup is complete, we recommend that you enable the integration with your Git provider to obtain the coverage status and summaries directly on pull requests.
```

Comment:
If this feature will be enabled by default, does this recommendation make sense?

Reply:
Both the GitHub status checks and the new coverage summary [will be on by default on new repositories](https://codacy.atlassian.net/browse/COV-22). :+1: But I was thinking about highlighting the feature for customers with existing repositories, or even for customers who may have turned off the feature by chance before they set up the coverage.

Indeed, the ROI of having this recommendation here should be low and it's probably better to keep the coverage setup page a little bit simpler. What are your thoughts @fjrdomingues?

Reply:
Following the flow of the documentation, and with "After validating that the coverage setup is complete". It sounded to me like that's the use case where a user is adding Coverage for the first time. Recommending the user enable the feature may be wasteful because, most likely, the feature will already be enabled. Either way, I think this is minor.

I'm more concerned with users knowing that they need to have quality settings enabled for coverage to receive summaries and status checks on the provider. Do you think that's coverage in the docs?

Reply:
> Recommending the user enable the feature may be wasteful because, most likely, the feature will already be enabled.

Good point, and I agree. I reverted the changes to the page [Adding coverage to your repository](https://feat-coverage-summaries-cov-16-cov--docs-codacy.netlify.app/coverage-reporter/). :heavy_check_mark: 

> I'm more concerned with users knowing that they need to have quality settings enabled for coverage to receive summaries and status checks on the provider. Do you think that's coverage in the docs?

Yes, for the status checks the users must set up coverage **and** configure quality gates for coverage, so [we already had a note here](https://docs.codacy.com/repositories-configure/integrations/github-integration/#status-checks). For the coverage summaries, users only need to set up coverage (and if they also have a quality gate configured they'll get the OK/NOK emojis). For that, [we're adding this note](https://github.com/codacy/docs/pull/1546/files#diff-7f4e89a3c87e75816c07be0a47a92090b7833df324af55e6fe259ed6e9b85e9eR63-R64). :heavy_check_mark: 

I'm going to mark this conversation as solved for now, but we'll have another chance to review everything end-to-end, already including the final screenshots, before we publish the changes.